src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -130,10 +140,10 @@ internal virtual MetadataReferenceResolver GetCommandLineMetadataReferenceResolv
         /// <param name="file">Source file information.</param>
         /// <param name="diagnostics">Storage for diagnostics.</param>
         /// <returns>File content or null on failure.</returns>
-        internal SourceText ReadFileContent(CommandLineSourceFile file, IList<DiagnosticInfo> diagnostics)
+        internal SourceText TryReadFileContent(CommandLineSourceFile file, IList<DiagnosticInfo> diagnostics)
```

Comment:
>TryReadFileContent [](start = 28, length = 18)

We sometimes use the `Opt` suffix for methods that may return `null` instead of using the `Try` prefix (which is more common with the bool/out pattern). I don't have strong feelings about this, though.